Makefile

```diff
@@ -50,6 +50,7 @@ test: export CM_DATABASE_URL=postgresql://cm-service@localhost:${CM_DATABASE_POR
 test: export CM_DATABASE_PASSWORD=INSECURE-PASSWORD
 test: export CM_DATABASE_SCHEMA=cm_service_test
 test: run-compose
+	cm-service init
```

Comment:
Was this required by something in particular?  The "engine" test fixture already has plumbing in place to reset/init the database (see https://github.com/lsst-dm/cm-service/blob/1b2864745af3dca11786a11873b43a6f7ef08c85/tests/conftest.py#L24)

Reply:
The test were failing with errors that suggested the database wasn't up and this fixed it.  I'll try removing it.